cli/src/test/resources/hudson/cli/dsa

```diff
@@ -0,0 +1,12 @@
+-----BEGIN DSA PRIVATE KEY-----
```

Comment:
Not sure if there's an easy pure-Java way to generate these keys, but using randomly generated keys each time the test is run would at least prevent security scanners from complaining about versioned secrets.

Reply:
There probably isn't a way to easily generate these for tests. Do security scanners look in test code for such things?

Reply:
Some do, yes. I updated a test in jsch-plugin to avoid the warning: https://github.com/jenkinsci/jsch-plugin/pull/6

Reply:
those keys are generated with ssh-keygen that it is probably the most common tool to make it. If I make it using Java, it would add a bunch of code not related to tests.